pydantic/main.py

```diff
@@ -74,6 +65,11 @@
 
 __all__ = 'BaseModel', 'create_model'
 
+# Keep these type aliases available at runtime:
+TupleGenerator: TypeAlias = Generator[Tuple[str, Any], None, None]
+# Keep this type alias in sync with the stub definition in `pydantic-core`:
+IncEx: TypeAlias = 'set[int] | set[str] | dict[int, IncEx | bool] | dict[str, IncEx | bool]'
```

Comment:
I removed the `| None` and added it in the relevant method signatures, as it makes more sense imo to keep the type alias scoped to what it represents, and then have consumers of this type alias make it optional or not.

Reply:
Good call.